evalml/utils/woodwork_utils.py

```diff
@@ -146,6 +153,26 @@ def downcast_nullable_types(X, ignore_null_cols=True):
     new_ltypes.update(
         {col: "Double" for col in X_int_nullable_cols if col in non_null_columns},
     )
+
+    if new_ltypes:
+        X.ww.set_types(logical_types=new_ltypes)
+    return X
+
+
+def downcast_int_nullable_to_double(X):
```

Comment:
Why add a separate function here? Isn't this just a subset of the `downcast_int_nullable_to_double` behavior?

Reply:
These two functions provide different use cases currently, I've filed an issue [here](https://github.com/alteryx/evalml/issues/3706)!